src/antsibull/cli/antsibull_docs.py

```diff
@@ -174,8 +174,10 @@ def parse_args(program_name: str, args: List[str]) -> argparse.Namespace:
 
     cache_parser = argparse.ArgumentParser(add_help=False)
     # TODO: Remove --ansible-base-cache once the ansible/ansible docs-build test is updated
-    cache_parser.add_argument('--ansible-base-source', '--ansible-base-cache', default=None,
-                              help='Checkout or expanded tarball of the ansible-base package.  If'
+    # TODO: Eventually remove --ansible-base-source
+    cache_parser.add_argument('--ansible-core-source', '--ansible-base-source',
+                              '--ansible-base-cache', default=None,
```

Comment:
Do we also need `--ansible-core-cache` here?

Reply:
No. The other names have been deprecated and will get removed eventually (except `--ansible-core-source`, which is the new final name to keep), so there's no need to add another thing to remove just for symmetry reasons :)